src/frontend/citizen-portal/src/app/shared/pipes/format-date.pipe.ts

```diff
@@ -9,7 +8,21 @@ import { APP_DATE_FORMAT } from '@shared/modules/ngx-material/ngx-material.modul
 export class FormatDatePipe implements PipeTransform {
   transform(date: string, format: string = APP_DATE_FORMAT): string {
     if (date) {
-      date = moment(date, 'YYYY-MM-DD').format(format);
+      const parts = date.split('-');
+      let newDate;
+      if (parts.length === 3) {
+        newDate = new Date(
```

Comment:
Should this hand invalid input, if any of the fields are non numeric, newDate will contain 'Invalid Date'

![image](https://user-images.githubusercontent.com/1844480/126355447-6f079e3c-f363-45f3-8e48-c9580a9a29a5.png)